packages/storybook/stories/inbox/full-page-inbox-hooks.tsx

```diff
@@ -28,6 +29,7 @@ export const FullPageInboxHooks: React.FunctionComponent = () => {
     >
       <h3>My Inbox</h3>
       <div>Unread Messages: {unreadMessageCount}</div>
+      <button onClick={() => markAllAsRead()}>Mark All Read</button>
```

Comment:
?
```suggestion
      <button onClick={markAllAsRead}>Mark All Read</button>
```

Reply:
Ohhh. Is it cause you don't want the event passed to the function? 

Disregard.